npm/src/lib/github.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import GitHub from "github";
+
+const CLIENT = new GitHub({version: "3.0.0"});
+if (process.env.GH_TOK) {
```

Comment:
No, it's optional. The only reason I added this is because the GitHub API will rate limit you more aggressively if you don't have a token -- so I added such a token to Travis to hopefully keep test runs from getting too flaky